.gitignore

```diff
@@ -1,27 +1,43 @@
+# Please keep workflow- and tool-specific patters out of this
+# file. Just put patterns here that are very generic and we want to
+# avoid adding by mistake and files that are generated by the build
+# system.
+# 
+# Comments from gitignore(1) mention that which file to place a
+# pattern in depends on how the pattern is meant to be used:
+#
+# * Patterns which should be version-controlled and distributed to
+#   other repositories via clone (i.e., files that all developers will
+#   want to ignore) should go into a .gitignore file.
+#
+# * Patterns which are specific to a particular repository but which
+#   do not need to be shared with other related repositories (e.g.,
+#   auxiliary files that live inside the repository but are specific
+#   to one user’s workflow) should go into the $GIT_DIR/info/exclude
+#   file.
+#
+# * Patterns which a user wants Git to ignore in all situations (e.g.,
```

Comment:
I think what this is trying to say is that if you'd like the ignore preferences to apply across all your repos (e.g., because of IDE preferences), you should use your global config.

However, I don't think this necessarily excludes doing it also on a per-repo basis (i.e., `.gitignore`) for common IDEs, in particular those that are common among our team.